### ISSUE TITLE
Questions about the implementation.

### ISSUE THREAD

**Issue by MaFuyan** [IoannisKansizoglou/Active-Emotion-Recognition#2]
Nice work. 
However, I didn't found the **confidence estimation part** and **REINFORCE algorithms** in this repository. It seems that you just implement the audio-visual emotion classification part. 
Would you mind making those parts open-sourced?
If I misunderstood something, please feel free to point it out.

**Comment by IoannisKansizoglou** [comment]
Hey, you 're totally right about that! 
The initial code of the paper has been developed in Tensorflow 1.7 (sadly not available at the moment) and now I'm gradually attempting to transfer it in Pytorch.
I 'm aiming for a final commit with the whole pipeline in December..
Thank you for your interest. Open for any question or discussion about the method.

**Comment by MaFuyan** [comment]
Aha~Thanks for your quick reply.
This work is pretty interesting, especially in HRI. I used to think how to use the change of emotions as a reward-penalty function to achieve natural HRI. But I didn't find it out :(
Although your work didn't provide an explicit solution to my problem, it is indeed a nice attempt to do that.
Look forward to your whole project.
(And maybe we can work together to produce an extended work. )
